specs/draggable.spec.jsx

```diff
@@ -59,6 +59,19 @@ describe('react-draggable', function () {
       assert(node.getAttribute('class') === 'foo react-draggable');
     });
 
+    it('should set the appropriate custom className when dragging or dragged', function () {
+      drag = TestUtils.renderIntoDocument(
+        <Draggable>
```

Comment:
Happy to make this change, but considering [this](https://github.com/mzabriskie/react-draggable/blob/master/lib/Draggable.es6#L137-L140) and [this](https://github.com/mzabriskie/react-draggable/blob/master/lib/DraggableCore.es6#L152-L155) it seemed inconsistent to require `className` to be a child property but `classNameDragged` and `classNameDragging` to be self properties.

I'm not sure why `className` was ever a child property anyway, but I didn't want to suggest a non-backwards-compatible Api change.